doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,83 @@
+can be scheduled onto the worker nodes. Each worker node runs helper processes which
+implement distributed scheduling and :ref:`memory management<memory>`.
 
-.. image:: ray-cluster.jpg
+.. figure:: ray-cluster.jpg
     :align: center
     :width: 600px
 
-.. _cluster-head-node-under-construction:
-
-Head node
-~~~~~~~~~
+    A Ray cluster consists of a set of one or more worker nodes, one of which is designated
+    as the head node. Each node runs Ray helper processes and Ray application code.
 
-The head node is the first node started by the
-:ref:`Ray cluster launcher<cluster-launcher>` when trying to launch a Ray
-cluster. Among other things, the head node holds the :ref:`Global Control Store
-(GCS)<memory>` and runs the :ref:`autoscaler<cluster-autoscaler>`. Once the head
-node is started, it will be responsible for launching any additional
-:ref:`worker nodes<cluster-worker-node>`. The head node itself will also execute
-tasks and actors to utilize its capacity.
+The number of worker nodes in a cluster may change with application demand, according
+to your Ray cluster configuration. The :ref:`head node<cluster-head-node-under-construction>`
+runs the logic which implements autoscaling.
 
-.. _cluster-worker-node-under-construction:
+.. note::
+    Here, a *node* means an individual machine. However, :ref:`on Kubernetes<kuberay-index>`, nodes are implemented as pods.
 
-Worker node
-~~~~~~~~~~~
-
-A worker node is any node in the Ray cluster that is not functioning as head node.
-Therefore, worker nodes are simply responsible for executing tasks and actors.
-When a worker node is launched, it will be given the address of the head node to
-form a cluster.
-
-.. _cluster-launcher-under-construction:
-
-Cluster launcher
-----------------
+.. _cluster-head-node-under-construction:
 
-The cluster launcher is a process responsible for bootstrapping the Ray cluster
-by launching the :ref:`head node<cluster-head-node>`. For more information on how
-to use the cluster launcher, refer to
-:ref:`cluster launcher CLI commands documentation<cluster-commands>` and the
-corresponding :ref:`documentation for the configuration file<cluster-config>`.
+Head node
+---------
+Every Ray cluster has one :ref:`worker node<cluster-worker-nodes-under-construction>`
+which is designated as the **head node** of the cluster. The head node runs
+important processes such as the :ref:`autoscaler<cluster-autoscaler-under-construction>`
+and the Ray driver processes, :ref:`which run the top-level Ray application
+<cluster-clients-and-jobs-under-construction>`. Ray may schedule tasks and actors
+on the head node just like any other worker node.
+
+.. note::
+    :ref:`On Kubernetes<kuberay-index>`, the autoscaler process runs inside a sidecar container in the head pod,
+    instead of in the same container (which would be most analogous to how the autoscaler runs inside the head node
+    on non-Kubernetes clusters).
```

Comment:
I think this wording is ok.
On VMs, we typically also recommend running in a Docker container, in which it case it is literally the case that the autoscaler runs in the same container as the rest of the Ray process.
(It is also possible to run on the bare VM with no Docker container.)